src/code42cli/cmds/departing_employee.py

```diff
@@ -15,7 +15,6 @@
 from code42cli.cmds.shared import get_user_id
 from code42cli.errors import Code42CLIError
 from code42cli.file_readers import read_csv_arg
-from code42cli.file_readers import read_flat_file_arg
```

Comment:
We can also delete this arg + function since they're no longer used anywhere.

Reply:
I mentioned it above but do you think it'd be fine to just remove all the methods related to the flat-files then?

Reply:
oops, missed that comment. yeah, let's remove them since the csv reader can handle the single column case just as well now.